### ISSUE TITLE
Arrows are missing from scrubbing slider when you drag it.

### ISSUE THREAD

**Issue by kevinb7** [Khan/live-editor#419]
Reported by @JavascriptFTW in https://github.com/Khan/live-editor/issues/415 in Firefox on Windows.  I was able to repro on Mac on Chrome.

**Comment by Gigabyte-Giant** [comment]
The arrows aren't missing, they just seem to move a bit oddly.
![scrubber-problem](https://cloud.githubusercontent.com/assets/4977819/9768539/f3a9c304-56e9-11e5-9549-1ba8868b750a.png)

Chrome 45.0.2454.85 on Mac OS X Yosemite.

**Comment by kevinb7** [comment]
You're right, I was confused because it's mostly obscured by the popup.  Looks like an ordering issue with the CSS.
<img width="297" alt="screen shot 2015-09-09 at 11 22 38 am" src="https://cloud.githubusercontent.com/assets/1044413/9770465/2e90aeba-56e5-11e5-9b57-4f38b53d9721.png">
